lib/perl/Genome/Model/Build/DeNovoAssembly/Allpaths.pm

```diff
@@ -139,7 +139,7 @@ sub before_assemble {
         my $lib_id = join("-", @lib_ids);
         my $genomic_start = 0;
         my $genomic_end = 0;
-        my $lib_name = join("-", map{Genome::Library->get($_)->name} @lib_ids);
+        my $lib_name = join("-", map{ my $name = Genome::Library->get($_)->name; Genome::Utility::Text::sanitize_string_for_filesystem($name); } @lib_ids);
```

Comment:
It is not being used as a file name. This is going into a "CSV" file that is read in by ALLPATHS. Unfortunately, the ALLPATHS CSV parsing code does not respect quotes, and adding them will not fix the problem.